OCCommunicationLib/OCCommunicationLib/OCOAuth2Authentication/OCOAuth2Manager.m

```diff
@@ -0,0 +1,134 @@
+//
+//  OCOAuth2Manager.m
+//  ownCloud iOS library
+//
+//  Created by Noelia Alvarez on 28/08/2017.
+//
+// Copyright (C) 2017, ownCloud GmbH.  ( http://www.owncloud.org/ )
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+#import "OCOAuth2Manager.h"
+
+@implementation OCOAuth2Manager
+
++ (void) getAuthDataByOAuth2Configuration:(OCOAuth2Configuration *)oauth2Configuration
```

Comment:
The ```get``` prefix is a bit confusing to me, specially because the method return ```void```. I would prefer something like ```refreshAuthDataByOAuth2Configuration```.

Reply:
👍